metpy/calc/basic.py

```diff
@@ -257,6 +257,77 @@ def pressure_to_height_std(pressure):
     return (t0 / gamma) * (1 - (pressure / p0).to('dimensionless')**(Rd * gamma / g))
 
 
+@exporter.export
+@check_units('[length]')
+def height_to_geopotential(height):
+    r"""Computes geopotential for a given height in meters
+
+    Parameters
+    ----------
+    height : `pint.Quantity`
+        Height above sea level (array_like)
+
+    Returns
+    -------
+    `pint.Quantity`
+        The corresponding geopotential value(s)
+
+    Examples
+    --------
+    >>> from metpy.constants import g,Re,G,me
+    >>> import metpy.calc
+    >>> from metpy.units import units
+    >>> height = np.linspace(0,10000, num = 11) * units.m
+    >>> geopot = metpy.calc.height_to_geopotential(height)
+    >>> geopot
+    <Quantity([     0.           9817.70342881  19632.32592389  29443.86893527
+    39252.33391207  49057.72230251  58860.0355539   68659.27511263
+    78455.4424242   88248.53893319  98038.56608327],
+    'meter ** 2 / second ** 2')>
+    """
```

Comment:
There should be a blank line at the end of the docstring (inside the quotes).

Reply:
Added the blank lines in the docstring(s).